tests/lib/openstack_query/query_blocks/conftest.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import MagicMock
+import pytest
```

Comment:
I think you can change this to import only fixture. Fixture is quite obvious about where it comes from since we only see it in a test file.
```suggestion
from pytest import fixture
```

Reply:
I'd rather not - I think `pytest.fixture` is more readable. Look up namespace pollution - https://stackoverflow.com/questions/22903542/what-is-namespace-pollution

where you draw the line is up to you, but "fixture" in my mind is ambiguous - and is more explicit when you do pytest.fixture in the code